mrtt-ui/src/components/CheckboxGroupMangroveWithLabel.js

```diff
@@ -22,16 +23,25 @@ const CheckboxGroupMangroveWithLabel = ({
       <Controller
         name={fieldName}
         control={formControl}
-        defaultValue={[]}
         render={({ field }) => {
+          const { otherValue, otherThanOtherSelectedValues } = separateOtherValueFromSelectedValues(
+            {
+              selectedValues: field.value,
+              options
+            }
+          )
+          const fieldToUse = { ...field, value: otherThanOtherSelectedValues }
           return (
```

Comment:
Im not sure about this. Generic checkbox group just gives back an array of strings, and doesnt differentiate the other. I made this util function if the parent wanted to separate. I think there must be a cleaner way, but my brain hasnt let me do it yet